.github/workflows/release.yaml

```diff
@@ -29,13 +29,10 @@ jobs:
           go-version: ${{ env.GO_VERSION }}
 
       - name: "Run GoReleaser"
-        uses: goreleaser/goreleaser-action@v2
-        with:
-          version: latest
-          args: release --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           HOMEBREW_GETENVOY_GITHUB_TOKEN: ${{ secrets.GETENVOY_CI_GITHUB_TOKEN }}
+        run: make release
```

Comment:
we now use bingo, so there will be exact match between the goreleaser we linted with and what we actually release with